src/event-handlers.js

```diff
@@ -27,13 +28,14 @@ const {
   CLEANUP,
   FERTILIZE,
   HARVEST,
+  MINE,
   PLANT,
   SET_SCARECROW,
   SET_SPRINKLER,
   WATER,
 } = fieldMode
 
-const toolbeltFieldModes = [CLEANUP, HARVEST, WATER]
+const toolbeltFieldModes = [CLEANUP, HARVEST, WATER, MINE] // todo: where can this data come from that is more dynamic? data/tools.js?
```

Comment:
at the time i was thinking about if it needed to consider the `features` or not, but i think this is kind of out of place in general too. maybe it fits in `enums.js`? we have `fieldMode` there already which this is a subset of, or maybe it fits with the `Toolbelt` component better? 

we're also manually controlling the calls to `handleFieldModeSelect`, so while this feels like it's an "allowlist" type of array, i think just checking against `fieldMode` from enums could do the same job, and not require extra maintenance. 

we don't really have to do anything with this for this PR, but let me know if any of that resonates with you (or doesn't make sense 😅 )

Reply:
I see what you're saying. I don't think it belongs in `enums.js` because it's a list and not an enum. However, I think it would make sense to move this to `constants.js`!

>we're also manually controlling the calls to handleFieldModeSelect, so while this feels like it's an "allowlist" type of array, i think just checking against fieldMode from enums could do the same job, and not require extra maintenance.

Yeah this is effectively an allowlist. I don't think we should reuse the `fieldModes` enum for this because `toolbeltFieldModes` is intentionally a subset of those values and we don't want to match with anything beyond it. 

Upon looking at this closer, `toolbeltFieldModes` should be a faster data structure (like an Object or a Set). Here's the only place it's used: https://github.com/jeremyckahn/farmhand/blob/2a4966b1c98fe62bf4c94d0be27e7757279328c2/src/event-handlers.js#L127-L135

I don't know why I went with an array for this initially (it's been a few years since I wrote this), but this is as good a time as any to improve it since this is the first time it has been touched in a while!

-------------------------

**tl;dr**: I'd say this is fine to leave as-is for this PR, but if you're interested in improving this then I'd support converting this to a  faster data structure and moving it to `constants.js`. 🙂

Reply:
added it as a Set to constants!